bin/test/ci.sh

```diff
@@ -48,13 +48,10 @@ make tests
 
 EXIT_STATUS=$?
 
-pkill iOSDeviceManager
-pkill Simulator
```

Comment:
I see these have been added to the `post`.

They also need to be run before starting the other tests.

Maybe add a "setup" node where iOSDeviceManager and Simulator are terminated?

Reply:
Yeah, fair enough.